Dockerfile

```diff
@@ -1,7 +1,7 @@
-ARG GO_VERSION=1.17
-FROM golang:${GO_VERSION}-alpine3.14
+ARG GO_VERSION=1.19
+FROM golang:${GO_VERSION}-alpine3.16
 
 LABEL org.opencontainers.image.source="https://github.com/Countingup/docker-go"
 
-RUN apk add --no-cache --update --upgrade git openssh-client make bash lftp coreutils zip curl libssl1.1 libcrypto1.1
+RUN apk add --no-cache --update --upgrade git openssh-client make bash lftp coreutils zip curl
```

Comment:
> removed update package command for libssl1.1 and libcrypto1.1

have these been updated in the base image?

Reply:
yeah. Ran the snyk test to make sure it was ok. The problem we were having was that golang was slow in rebuilding their image but this should be the case now that we are using alpine3.16 (people usually tend to update the image with latest alpine more often)